packages/accordion/src/js/jquery.js

```diff
@@ -0,0 +1,86 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * Accordion function
+ *
+ * A component to allow users to show or hide page elements.
+ *
+ **************************************************************************************************************************************************************/
+
+
+/**
+ * The accordion jquery function to add attributes and event listeners
+ *
+ * Public functions are:
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().isOpen()
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().setA11y()
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().addListeners()
+ */
+$.fn.uikitAccordion = function() {
+	var $elements = this;
+
+	return {
+		/**
+		 * Check an accordion for its state
+		 *
+		 * @return {Boolean} - true = open; false = closed
+		 */
+		isOpen: function() {
+			return !$elements.children('.js-uikit-accordion').hasClass('uikit-accordion--closed');
+		},
+
+
+		/**
+		 * Adding the correct aria and role attributes
+		 */
+		setA11y: function() {
+			$elements.each( function( i, accordion ) { //render multiple elements
+				var $accordion = $( accordion );
+				var _isOpen = $accordion.uikitAccordion().isOpen();
+				var $controls = $accordion.children('.js-uikit-accordion');
+				var $body = $accordion.children('.js-uikit-accordion-body');
```

Comment:
`.children` is more specific than `.find` so it should be faster. That's my understanding.